src/components/navigation/EdgeLogoHeader.tsx

```diff
@@ -8,7 +9,7 @@ export function EdgeLogoHeader() {
   const styles = getStyles(theme)
   return (
     <View style={styles.container}>
-      <Image style={styles.icon} source={theme.headerIcon} resizeMode="contain" />
+      <FastImage style={styles.icon} source={theme.headerIcon} resizeMode="contain" accessibilityHint="Edge Logo Header" />
```

Comment:
The `accessibilityHint` should be a translated string, and it can't say "Edge", because white-labels again. Maybe add `lStrings.logo_hint: "App logo"`

Reply:
If we are treating `accessibilityHint`'s as first-class citizen strings then https://github.com/EdgeApp/edge-login-ui-rn/pull/42 needs to be fixed too.

Reply:
Looks like we are, so this string has been moved to it's own 'section' in [`cdb6412` (#4131)](https://github.com/EdgeApp/edge-react-gui/pull/4131/commits/cdb64129d285558c969d0ec97de46546473c72d3)